wikipendium/wiki/views.py

```diff
@@ -107,7 +109,15 @@ def add_tag_to_article(request, slug):
     if 'tag' not in request.POST:
         return HttpResponseBadRequest()
 
-    tag = slugify(request.POST['tag'])
+    slug = original_slug = slugify(request.POST['tag'])
+    for i in itertools.count(1):
```

Comment:
That's the way it was done where I got the idea: https://keyerror.com/blog/automatically-generating-unique-slugs-in-django#ensuring-uniqueness